docs/system_overview/data_objects.md

```diff
@@ -266,27 +267,46 @@ Trim refers to unlinking one or more replicas mapped to an existing data object.
 
 | Case | Replica A start | Replica B start | Replica A end | Replica B end | Reason |
 | ---- | --------------- | --------------- | ------------- | ------------- | ------ |
 | 0 | - | - | - | - | Data object does not exist |
 | 1 | - | & | - | & | Data object has fewer than two replicas |
 | 2 | - | X | - | X | Data object has fewer than two replicas |
 | 3 | & | - | & | - | Data object has fewer than two replicas |
-| 4 | & | & | - | & | Trim is allowed\* |
+| 4 | & | & | - | & | Trim is allowed |
 | 5 | & | X | & | - | Trim is allowed |
 | 6 | X | - | X | - | Data object has fewer than two replicas |
 | 7 | X | & | - | & | Trim is allowed |
-| 8 | X | X | X | X | Data object has fewer than the minimum number of good replicas |
-
-\* The oldest repilca is trimmed in case 4
+| 8 | X | X | - | X | Trim is allowed |
+
+In order to understand the priority order of how replicas are selected for trimming, we now include a more complicated table. The table shows 4 characters which represent 4 replicas. The replicas are ordered by modify time, from least recent to most recent (that is, from oldest to newest). As with the previous table, 'X' represents a stale replica, '&' represents a good replica, and '-' indicates a replica which does not exist. The table shows the starting and ending statuses of the 4 replicas for a single data object such that the status of the oldest replica is the first character, the status of the second-oldest replica is the second character, the status of the second-newest replica is the third character, and the status of the newest replica is the fourth character. The ending statuses represent what happens when trim is executed on the data object without selecting any replicas in particular and using the default of two minimum number of replicas to retain.
+
+| Case | start | end | Reason |
+| ---- | ---- | ---- | ---- |
+| 0 | XXXX | --XX | Oldest stale replicas are trimmed, down to minimum number of replicas. |
+| 1 | XXX& | --X& | Oldest stale replicas are trimmed, down to minimum number of replicas. |
+| 2 | XX&X | --&X | Oldest stale replicas are trimmed, down to minimum number of replicas. |
+| 3 | XX&& | --&& | All stale replicas are trimmed. No good replicas are trimmed because minimum number of replicas has been reached. |
+| 4 | X&XX | -&-X | Oldest stale replicas are trimmed, even though the good replica is older than one of the trimmed stale replicas. |
+| 5 | X&X& | -&-& | All stale replicas are trimmed, even though one of the good replica is older than one of the trimmed stale replicas. No good replicas are trimmed because minimum number of replicas has been reached. |
```

Comment:
good replica is older -> good replicas is older

Reply:
Done